components/console-backend-service/Dockerfile

```diff
@@ -1,43 +1,52 @@
-FROM golang:1.11.4-alpine3.8 as builder
+FROM golang:1.13.11-alpine3.11 as builder
 
 ENV BASE_APP_DIR /go/src/github.com/kyma-project/kyma/components/console-backend-service
+
+ENV GO111MODULE on
+ENV CGO_ENABLED 0
+
+
 WORKDIR ${BASE_APP_DIR}
 
-#
-# Copy files
-#
+COPY ./go.mod .
+COPY ./go.sum .
+
+COPY ./internal ./internal
+COPY ./pkg ./pkg
+COPY ./main.go .
+COPY ./Makefile .
+
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
```

Comment:
`go mod download` must be run before any sources are copied. Otherwise caching won't work